public/js/apps/management/app.jsx

```diff
@@ -56,7 +59,10 @@ export default class ManagementApp extends Component {
       ));
     }
 
-    children.push(<Management {...boundMgmtActions} {...boundUserActions}
+    children.push(<Management {...boundMgmtActions}
+                              {...boundUserActions}
+                              {...boundSubscriptionActions}
```

Comment:
Hah, I'm adding one here too - I wonder if we can merge the actions into one. `bindActionCreators` could do with taking a list.